deploy/crd.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cspcautos.dao.mayadata.io
```

Comment:
What is this for?

Reply:
These are the older designs. CSPCAuto is now `CStorPoolConfig` with new specifications.
You may only refer to `design.md` for the time being.